### ISSUE TITLE
General Discussion

### ISSUE THREAD

**Issue by housseindjirdeh** [GoogleChrome/lighthouse-platform-packs#3]
Opening this issue to have a place where people can leave general suggestions 💬.

Some preliminary thoughts to kick things off:

* Which platforms should we support?
  * After WordPress, we can continue to target other CMS platforms (like Drupal) as well as explore how to include suggestions for JavaScript frameworks (Angular, React, etc...). Anything else?
* Should we show a toggle to switch between the original Lighthouse report and an updated one with platform-specific strings included? Or should we always surface platform-specific audit messages if a tool is detected on the webpage?

_Note: For details on how platform packs will be integrated into core, take a look at the [RFC](https://github.com/GoogleChrome/lighthouse/issues/7021)._

**Comment by tpiros** [comment]
Not a platform/framework per se but would love to see more detailed report about images (or even media) in general.

**Comment by docluv** [comment]
This looks like you are just adding secondary messages based on the platform right now. Not really adding tests at the moment.
Are you planning on adding additional tests/rules for a CMS?
Plus how do you plan on actually auditing server-side code in a platform like WordPress?

Since I use my own platform, can I use this as a model to create my own customized report or an I better off just running it using a node module and then rendering the JSON against my own template. Seems like the same thing you are going right now.

**Comment by haassie** [comment]
I think the opensource CMS TYPO3 might be an interesting next platform. It is already the cms with the highest density of fast FCP as you can see on https://discuss.httparchive.org/t/cms-performance/1468. Besides that the CMS and the community is really having performance as one of the most important topics. So I will definitely go for TYPO3 as one of the next platforms!

**Comment by housseindjirdeh** [comment]
@docluv Yep, that is correct. We're only planning to add secondary messages for already existing audits to make them more platform-specific. We're not planning to add any extra audits for any tool.

We don't plan to do anything fancy to audit server-side code. We're planning to rely on audits that are already available and flag common issues that folks who use WordPress are most likely experiencing.

If you wanted to set up a report of your own for a specific tool, you might just have to dive into the audit files and include additional messages yourself. You can then boot up the report with Lighthouse Viewer like I did with the [first prototype](https://github.com/housseindjirdeh/lighthouse/commit/13d3d28a4d5a8fff42604d51dc69332c8587b7b4). Once the feature is fully developed however, we may have a better way to fork and build your own version.

**Comment by housseindjirdeh** [comment]
@haassie I created a separate issue (#7) to list down different tools we can work towards supporting :)

**Comment by ericerway** [comment]
Nice work! Would love to see this for Magento after Wordpress. We're doing some great things around PWA as a commerce platform with our community. /cc @zetlen

**Comment by ezekielchentnik** [comment]
Not framework specific but would love to see more details and reporting for WebGL specific stuff;  three.js could also be a framework to focus on.

**Comment by housseindjirdeh** [comment]
@ericerway We've already been chatting about supporting Magento :) We'll most definitely keep you in the loop when we begin rolling things out for more than just WordPress.

CC @rowan-m

**Comment by beanstalkim** [comment]
OK ... trying to install for the first time and making what I can only assume is some stupid issue.

I keep getting a missing manifest file error.

Hoping someone can help me remedy this PEBKAC scenario.

Thanks !

**Comment by housseindjirdeh** [comment]
Hi @beanstalkim! Are you just trying to install the library itself as an npm dependency?
